deps/libmediasoupclient/include/Logger.hpp

```diff
@@ -52,7 +52,7 @@ namespace mediasoupclient
 	class Logger
 	{
 	public:
-		enum class LogLevel : uint8_t
```

Comment:
no `libmediasoupclient` changes here please. Those need to be done in the corresponding project. This is just a demo project using it.

I don't think this is a needed changed by no means.

Reply:
Without the change, I cannot build:

Logger.hpp:55:25: error: ‘uint8_t’ was not declared in this scope
   enum class LogLevel : uint8_t

Reply:
Anyway, I'll change it just on my machine. I'll undo the changes in libmediasoupclient in this project.